sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractSummaryOptions.cs

```diff
@@ -4,14 +4,13 @@
 namespace Azure.AI.TextAnalytics
 {
     /// <summary>
-    /// Options that allow callers to specify details about how the operation
-    /// is run and what information is returned from it by the service.
+    /// A set of options used to configure extractive summarization, including the display name to use, the maximum
```

Comment:
Nit:
```suggestion
    /// A set of options used to configure extractive summarization including the display name to use, the maximum
```

Reply:
I think the comma is correct in this case because everything that comes after it could technically be removed without changing the meaning.